src/super_gradients/common/factories/transforms_factory.py

```diff
@@ -74,6 +75,8 @@ def _check_bbox_params(self, bbox_params):
             raise ValueError(f"Unexpected fixed bbox_params keys: {fixed_keys}. Fixed bbox_params {self.FIXED_BBOX_PARAMS} cannot be overriden.")
 
     def _check_keypoint_params(self, keypoint_params):
+        if len(keypoint_params.keys()):
+            raise ValueError("`keypoint_params` should be left empty. Please leave it as `{keypoint_params: None}` for .json or `keypoint_params: ` for .yaml")
```

Comment:
If the user sets any `keypoint_params` the code could crash with weird errors. There is no relevant params for users to set.
At the same time, we still need to know that the user is using keypoints, so the trick is to ask to the user to have `keypoint_params` but leave it empty. Not great, but couldnt find a better way after playing around for quiet some time